app/Exceptions/ReverseProxyNotConfiguredException.php

```
@@ -0,0 +1,26 @@
+    {
+        $message = $this->getMessage() !== ''
+            ? $this->getMessage()
+            : 'Please set a port and a start command before deploying this site.';
+
+        if ($request->header('X-Inertia')) {
+            return back()->with('error', $message);
+        }
+
+        throw ValidationException::withMessages([
+            'deployment' => $message,
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Validation\ValidationException' in method 'render'. (undefined)

(StaticAccess)